tests/005-test-volume_mount.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+job_name=volume_mount
+host_volume_path=/tmp/host_volume/s1
+
+# test volume_mount
+test_volume_mount_nomad_job() {
+    pushd ~/go/src/github.com/Roblox/nomad-driver-containerd/example
+
+    setup_bind_source
+
+    echo "INFO: Starting nomad $job_name job using nomad-driver-containerd."
+    nomad job run $job_name.nomad
+
+    # Even though $(nomad job status) reports job status as "running"
+    # The actual container process might not be running yet.
+    # We need to wait for actual container to start running before trying exec.
+    echo "INFO: Wait for ${job_name} container to get into RUNNING state, before trying exec."
+    is_${job_name}_container_active
+
+    echo "INFO: Checking status of $job_name job."
+    job_status=$(nomad job status -short $job_name|grep Status|awk '{split($0,a,"="); print a[2]}'|tr -d ' ')
+    if [ "$job_status" != "running" ];then
+        echo "ERROR: Error in getting ${job_name} job status."
+        exit 1
+    fi
+
+    # Check if bind mount exists.
+    echo "INFO: Checking if bind mount exists."
+    for mountpoint in t1 read_only_target ; do
+        output=$(nomad alloc exec -job ${job_name} cat /tmp/${mountpoint}/bind.txt)
+        if [ "$output" != "hello" ]; then
+           echo "ERROR: bind mount /tmp/${mountpoint} does not exist in container rootfs."
+           exit 1
+        fi
+    done
+
+    # Check read only mount can not write.
+    echo "INFO: Checking read only mount is not writable."
+    nomad alloc exec -job ${job_name} touch /tmp/read_only_target/writable_test.txt
```

Comment:
nitpick: Can we suppress this error message `touch: cannot touch '/tmp/read_only_target/writable_test.txt': Read-only file system` from the logs?

Maybe we can use the return code (rc) in the `if` condition. Something like [`this`](https://github.com/Roblox/nomad-driver-containerd/blob/master/tests/004-test-privileged.sh#L51-L56)

Reply:
Sorry, my bad, the `if` condition is correct, and should be based on the existence of the file.
However, Can we suppress the message though?

Reply:
output is redirect to /dev/null now.

```
+    nomad alloc exec -job ${job_name} touch /tmp/read_only_target/writable_test.txt &>/dev/null
```